test/framework/toy_build.py

```diff
@@ -1126,6 +1128,27 @@ def test_toy_sanity_check_commands(self):
 
         self.assertTrue(os.path.exists(toy_modfile))
 
+    def test_toy_add_easyconfig(self):
+        """ Test dumping of file in eb_filerepo in both .eb and .yeb format """
+        filename = 'toy-0.0'
+        test_ecs_dir = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'easyconfigs')
+        paths = [os.path.join(test_ecs_dir, '%s.eb' % filename), os.path.join(test_ecs_dir, 'yeb', '%s.yeb' % filename)]
+        ext = [EB_FORMAT_EXTENSION, YEB_FORMAT_EXTENSION]
```

Comment:
this will go out of sync easily, why not use `ext = os.path.splitext(path)[-1]` instead?